models/intermediate/int__location.sql

```diff
@@ -10,45 +9,31 @@
 WITH unioned_location_sources AS (
     SELECT DISTINCT
         p.patient_address AS address_1
-        , {{ dbt.cast("null", api.Column.translate_type("varchar")) }} AS address_2
         , p.patient_city AS city
         , s.state_abbreviation AS state
         , p.patient_zip AS zip
         , p.patient_county AS county
-        , {{ dbt.cast("null", api.Column.translate_type("integer")) }} AS country_concept_id
-        , {{ dbt.cast("null", api.Column.translate_type("varchar")) }} AS country_source_value
-        , p.patient_latitude AS latitude
-        , p.patient_longitude AS longitude
```

Comment:
I ran into fanning issues related to the fact that we were including lat/long in the location model, but that location_source_value (our join key from person to location) was only based on the address. I felt it best just to exclude these columns as most sources will not include this data.